frontend-cert/css-projects/newspaper-article/user-stories.md

```diff
@@ -0,0 +1,12 @@
+1. You should set the root `font-size` of your HTML document to `24px`.
+2. You should have an element with a class of `newspaper` that contains all your other elements.
+3. Your `.newspaper` element should have a `font-size` of `16px` and a font of `Open Sans` with a fallback font of `sans-serif`.
+4. Within your `.newspaper` element, you should have at least seven more elements. One for the newspaper name that has a class of `name`, one for the the date of the article with a class of `date`, one for the headline with a class of `headline`, one for the sub-headline with a class of `sub-headline`, one for the author with a class of `author`, and two paragraphs each with a class of `text`.
+5. Your `.name` element should have a `font-size` that's two times the size of the root elements `font-size` and a font of `Times New Roman` with a fallback font of `serif`.
+6. Your `.name` and `.author` elements should use CSS to make all its characters uppercase.
```

Comment:
I couldn't figure out if `text-transform` has been taught at this point.
I couldn't find it in the nutrition label.

Here are the lecture contents according to the sheet:

- Why should you use relative units for font size?
- What are web safe fonts?
- Why is it best practice to include a fallback value for font-family?
- What is the @font-face at rule and how does it work?
- How to work with external fonts like Font Squirrel and Google Fonts.
- How multiple fonts can increase page download size.

Reply:
I think it should be fine. It looks like it's covered in the flexbox workshop, which is right before the nutrition label.